src/bindings/c/include/openvino/c/ov_tensor.h

```diff
@@ -24,6 +24,38 @@ typedef struct ov_tensor ov_tensor_t;
  * @{
  */
 
+/**
+ * @struct ov_tensor_list_t
+ * @brief A list of tensors
+ */
+typedef struct ov_tensor_list ov_tensor_list_t;
+
+/**
+ * @brief Creat a tensor list
+ * @ingroup tensor
+ * @param tensors A point to ov_tensor_list_t
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_tensor_list_create(ov_tensor_list_t** tensors);
+
+/**
+ * @brief Add a tensor to a list
+ * @ingroup tensor
+ * @param tensor A point to the tensor will be added to tensor list
+ * @param tensors A point to tensor list
+ * @return Status code of the operation: OK(0) for success.
+ */
+OPENVINO_C_API(ov_status_e)
+ov_tensor_list_add(ov_tensor_list_t* tensors, const ov_tensor_t* tensor);
+
```

Comment:
Do we need a helper function to get some info of ov_tensor_list_t, such as get tensor number?